### DISCUSSION TITLE
Format toggle error?

### DISCUSSION THREAD

**Discussion by kapral18** [LazyVim/LazyVim#1634] [Q&A] [answered]
When I hit `<leader>uf` to toggle the formatting I get an error:

![CleanShot 2023-10-08 at 14 53 21@2x](https://github.com/LazyVim/LazyVim/assets/1625373/b061d82b-7576-4884-85e9-5f5e3161d048)

I assume it's a bug but not sure

my config: https://github.com/kapral18/config.nvim/blob/f64e0120596a01c05fdcbd3ad2f24baf230e3b0b/lua/config/lazy.lua

**Comment by folke** [comment]
Did you disable lspconfig?
Does this happen with autoformat? Or manual format?
Is lsp loaded in that buffer?

**Reply by kapral18** [reply]
I didn't disable anything specifically. All I did was the following.

1. Open nvim in an empty folder.
2. Start writing in the unnamed buffer and then save it with `:w run.lua`
3. Hit `<leader>uf`
4. Get this error

If I close nvim and reopen it starts working as expected.

**Reply by folke** [reply, marked as answer]
Fixed in https://github.com/LazyVim/LazyVim/commit/e11a3cbe800a1216062f7cefd25827f6fdad9daa

Will make a new release later today

**Reply by kapral18** [reply]
Epic! Thanks.
